### ISSUE TITLE
Mod disables artifact of delusion

### ISSUE THREAD

**Issue by kg-csl** [Cercain/ImprovedCommandEssence#17]
With this mod active, the new delusion artifact does nothing. Is this intended?

**Comment by Cercain** [comment]
Can you check if this is the case with base Command?

**Comment by kg-csl** [comment]
Delusion is fine with base Command. It might be a certain setting though. I'll attach my config.


```

[ImprovedCommandEssence]

## Set the amount of Common (white) options shown when opening a Command Essences.
# Setting type: Int32
# Default value: 6
itemAmountCommon = 9

## Set the amount of Uncommon (Green) options shown when opening a Command Essences.
# Setting type: Int32
# Default value: 4
itemAmountUncommon = 9

## Set the amount of Legendary (Yellow) options shown when opening a Command Essences.
# Setting type: Int32
# Default value: 2
itemAmountLegendary = 6

## Set the amount of Yellow items shown when opening a Command Essences.
# Setting type: Int32
# Default value: 2
itemAmountBoss = 4

## Set the amount of Blue items shown when opening a Command Essences.
# Setting type: Int32
# Default value: 2
itemAmountLunar = 1000

## Set the amount of Blue items shown when opening a Command Essences.
# Setting type: Int32
# Default value: 2
itemAmountVoid = 1000

## Set the amount of Orange items shown when opening a Command Essences.
# Setting type: Int32
# Default value: 4
itemAmountEquip = 9

## Set the amount of items shown when opening a Void Potential from a Void Cache.
# Setting type: Int32
# Default value: 6
itemAmountPotential = 1000

## Set the amount of items shown when opening a Void Potential from a Void Triple.
# Setting type: Int32
# Default value: 4
itemAmountPotentialCache = 1000

## Set the chance for item drops to drop as a Command Essence (0-100) from non-explicit chests
# Setting type: Int32
# Default value: 100
essenceChance = 100

## Set if category chests only show items of the corresponding category.
# Setting type: Boolean
# Default value: true
keepCategory = false

## Set if the Command Artifact is turn on in the Bazaar.
# Setting type: Boolean
# Default value: false
onInBazaar = true

## Set if the Command Artifact is turn on for Drop Ship items.
# Setting type: Boolean
# Default value: false
onInDropShip = false

## Set if the Command Artifact is turn on for Adaptive Chest items.
# Setting type: Boolean
# Default value: false
onForAdaptive = true

## Set if the Command Artifact is turn on for Void Potentials and Void Caches.
# Setting type: Boolean
# Default value: false
onForPotential = true

## Set if the Command Essences that drop from the Teleporter boss give the same options.
# Setting type: Boolean
# Default value: true
sameBossDrops = true

## Set if the item dropped by bosses killed via Trophy Hunter's Tricorn drop as a Command Essence (true) or the boss item (false)
# Setting type: Boolean
# Default value: false
onForTrophy = false

## Set if Scrappers spawn
# Setting type: Boolean
# Default value: false
enableScrappers = true

## Set if Printers spawn (onInDropShip must be on to drop as the item)
# Setting type: Boolean
# Default value: false
enablePrinters = false

## Set if Multishops spawn (onInDropShip must be on to drop as the item)
# Setting type: Boolean
# Default value: false
enableTerminals = true

## Set if Scrappers drop scrap (false) or Command Essence (true)
# Setting type: Boolean
# Default value: false
scrappersDropEssence = true

## Add item names to make them drop as the item not an essence (Can be used to force compatability for other mods with set drops ie. 'ItemIndex.idSagesBook')(comma seperated)
# Setting type: String
# Default value: 
customCompatibility =


```

**Comment by Cercain** [comment]
Ok, I think I've found the issue, 1.2.4.4 changed an API I was hooking onto, so it now ignores the Delusion stuff they added, looking to add it back in but the community package doesnt have the new API added yet

**Comment by kg-csl** [comment]
Awesome, glad you found the issue. And it's fine that the package doesn't
have the API yet because it really isn't that huge of a deal, people who
use this mod are likely not using Delusion because it's so overpowered.

**Comment by Cercain** [comment]
So I hadn't updated MMHOOK, got it all working now. will be fixed in 1.3.6 I'm uploading now

**Comment by kg-csl** [comment]
Amazing bug fixing service, thanks!
